src/app/frontend/common/appconfig/appconfig_service.js

```diff
@@ -44,4 +44,12 @@ export class AppConfigService {
       return null;
     }
   }
+
+  getDashboardVersion() {
```

Comment:
I think app config should be a good place to store these values. We also could extract it to a new service but I don't have any preferences here.

@maciaszczykm what do you think?